subtests/docker_cli/run_cgroup_parent/run_cgroup_parent.py

```diff
@@ -117,7 +117,10 @@ def _read_cid(self):
         Read container ID from --cidfile file, so we can replace {cid}
         in format string and do an exact match on expected values.
         """
-        cid = open(self.sub_stuff['cidfile'], 'rb').read().strip()
+        try:
+            cid = open(self.sub_stuff['cidfile'], 'rb').read().strip()
+        except IOError:
+            return "<n/a>"         # no cidfile, e.g. because of bad command
         if len(cid) < 12:
             raise DockerTestError("bad cid (length < 12) in --cidfile")
         return cid
```

Comment:
LGTM.  Worth adding a config./test to negativeusage for the invalid ``--cgroup-parent``?